corehq/apps/export/views/new.py

```diff
@@ -310,6 +307,8 @@ def get_empty_export_schema(self, domain, identifier):
 
     @property
     def _is_bulk_export(self):
+        if self.export_type is not CASE_EXPORT:
```

Comment:
nit: If you rename this method to "_is_bulk_case_export", that would be super and would look well here as well

https://github.com/dimagi/commcare-hq/pull/34281/files#diff-071e1abe1d05962760181bd51e2617e2f21833e813c28e5062f7629bc09dcbdfR119